manifests/server.pp

```diff
@@ -115,6 +119,11 @@
   }
   validate_string($package_name)
   validate_absolute_path($pid_file)
+  if $ppolicy {
+    validate_re($pp_hash_cleartext, '^TRUE$|^FALSE$')
+    validate_re($pp_use_lockout, '^TRUE$|^FALSE$')
+    validate_re($pp_forward_updates, '^TRUE$|^FALSE$')
```

Comment:
I would prefer these to be exposed as native booleans and do the conversion of `true -> 'TRUE'` & `false -> 'FALSE'` within the module, (maybe add a function to do that?). I'd also rather not abbreviate the `ppolicy` to `pp` in the parameter names, yeah they're long and wordy but it matches the other parameters. I also see 'pp' and think 'pretty print' 😄